docs/csharp/tutorials/exploration/csharp-6.yml

```diff
@@ -0,0 +1,249 @@
+
+    ```csharp
+    WriteLine($"The average word length is: {wordLength.Average():F2}");
+    ```
+
+- title: Quick and easy null checks
+  durationInMinutes: 2
+  content: |
+   The `?.` operator makes it easier to write logic that takes `null` values into account seamlessly, without extra `if` checks. To explore the feature, start by copying the following code into the interactive window to try it:
+
+   ```csharp
+   using System;
+
+   public class Program
+   {
+       public static void Main()
+       {
+         string s = null;
+         Console.WriteLine(s.Length);
+       }
+   }
+   ```
+
+   It throws a <xref:System.NullReferenceException>. Change the `.` member access to the **null conditional operator**:
+
+   ```csharp
+   Console.WriteLine(s?.Length);
+   ``` 
+
+   After this change, there's no output. That's because the result of `s?.Length` is a `int?` whose value in this example is `null`.  The `?.` returns `null` if its left operand is `null`. If the type of the right operand is a value type, the `?.` operator returns a nullable type for that type. In addition to `?.` you can use `?[]` for array or indexer access. Try the following code in the interactive window:
+
+    ```csharp
+    char? c = s?[0];
+    Console.WriteLine(c.HasValue);
+    ```
+
+    The null conditional operator composes, making it easy to avoid nested if clauses to access members of members. For example, try the following code in the interactive window:
+
+    ```csharp
+    string s = null;
+    bool? hasMore = s?.ToCharArray()?.GetEnumerator()?.MoveNext();
+    Console.WriteLine(hasMore.HasValue);
+    ```
+
+    The preceding example can be simplified by using the **null coalescing operator** to provide a default value:
+
+    ```csharp
+    bool hasMore = s?.ToCharArray()?.GetEnumerator()?.MoveNext() ?? false;
+    Console.WriteLine(hasMore);
+    ```
+
+    The **null conditional operator** helps you write code that makes your core logic clear while seamlessly testing for `null` values.
+
+- title: Exception filers
+  durationInMinutes: 2
+  content: |
+    Exception filters enable you to catch an exception based on some condition. A typical use is to create a filter method that logs exceptions and always returns false. Try the following code in the interactive window: It logs the exception type and message to the console. It returns false which indicates that the exception can't be handled. Try the program in the interactive window.
+
+    ```csharp
+    using System;
+
+    public class Program
+    {
+        public static void Main()
+        {
+            try 
+            {
+                string s = null;
+                Console.WriteLine(s.Length);
+
+                bool? hasMore = s?.ToCharArray()?.GetEnumerator()?.MoveNext();
```

Comment:
I did remove those two lines.

I like this example because it shows a nice use case where the exception filter always returns false. 

I want a good example that looks at a property or field of an exception, but I haven't found a concise one yet.